.github/workflows/2-deploy-to-staging.yml

```diff
@@ -13,13 +13,35 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/setup
       - run: yarn prepare
+      - name: Send Slack message that build is starting
+        id: slack
+        uses: slackapi/slack-github-action@007b2c3c751a190b6f0f040e47ed024deaa72844
+        with:
+          payload: |
+            {
+              "text": "Staging build started for branch: ${{ github.ref_name }}"
+            }
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
       - run: yarn build
         env:
           REACT_APP_STAGING: 1
       - name: Setup node@16 (required by Cloudflare Pages)
         uses: actions/setup-node@v3
         with:
           node-version: 16
+      - name: Send Slack message that deploy is starting
+        id: slack
+        uses: slackapi/slack-github-action@007b2c3c751a190b6f0f040e47ed024deaa72844
+        with:
+          payload: |
+            {
+              "text": "Starting staging Cloudflare pages deploy for branch: ${{ github.ref_name }}"
+            }
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
```

Comment:
Rather than send a slack message at start/deploy/deployed, I think it'd be more useful to send a slack message at start/success/fail.

I also worry this will introduce message fatigue, so an alternative would be just success/fail.